dspace/dspace.dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM janx/ubuntu-dev
+RUN sudo apt-get update -q \
```

Comment:
Nit: Please add a comment before this line, e.g. `# Install DSpace dependencies.`.